paper/paper.md

```diff
@@ -1,3 +1,4 @@
+Statement of need
```

Comment:
Hi @soodoku , thanks for the editing! Just a quick question, can you confirm if this line should be here? After I compiled your edited version of the paper
[mixR paper draft (edited).pdf](https://github.com/GaryBAYLOR/mixR/files/7799877/mixR.paper.draft.edited.pdf)
 I see that the first line seems redundant, and the title section does not compile correctly. The references are also not showing.

Thanks!